src/test/ui/issues/issue-41139.rs

```diff
@@ -8,5 +8,5 @@ fn main() {
     // This isn't great. The issue here is that `dyn Trait` is not sized, so
     // `dyn Fn() -> dyn Trait` is not well-formed.
     let t: &dyn Trait = &get_function()();
-    //~^ ERROR expected function, found `&dyn Fn() -> (dyn Trait + 'static)`
+    //~^ ERROR expected function, found `&dyn Fn() -> dyn Trait`
```

Comment:
Why did this change? Not a huge deal, but would be nice to know why

Reply:
This is a bug.

Reply:
Wait, do you mean the existing test is a bug? Or the new output is?

Reply:
The new output is buggy.